docs/documentation/platform/secret-sharing.mdx

```diff
@@ -1,37 +1,42 @@
 ---
 title: "Secret Sharing"
 sidebarTitle: "Secret Sharing"
-description: "Learn how to share time-bound secrets securely with anyone on the internet."
+description: "Learn how to share time or view-count bound secrets securely with anyone on the internet."
```

Comment:
I think users should be able to use both restrictions?

Reply:
I found it weird to have to setup both hence felt the UX might be better off to just let them choose the option and set only that.

Reply:
For reference, bitwarden does it like the following 
![Uploading image.png…]()